book/src/volume-expansion.md

```diff
@@ -0,0 +1,46 @@
+# Volume Expansion
+
+**Status:** Alpha
```

Comment:
If you make it a header, make sure to add another header below `Description` or `Overview`. The rawblock on doesn't do that and it's confusing because status and overview blend together in a single section.